streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -389,7 +380,11 @@ public void closeClean() {
 
     @Override
     public void closeDirty() {
-        close(false);
+        try {
+            close(false);
```

Comment:
Ditto here, if we have swallowed all possible exception inside `close`, then it is unnecessary to have another swallow here.